FlowCytometryTools/core/bases.py

```diff
@@ -841,9 +842,15 @@ def _get_ID2position_mapper(self, parser):
             parser = lambda x: parser[x]
         elif parser == 'name':
             parser = lambda x: (x[0], int(x[1:]))
-        elif parser == 'number':
+        elif parser.startswith('number'):
```

Comment:
how about calling it 'number_col_first' to make it more explicit that it iterates by columns first? I can rename the other one to be 'number_row_first'.